README.md

```diff
@@ -7,60 +7,221 @@ List of availables languages:
 - [English](./README.md)
 - [Français](./README-fr.md)
 
+<br />
+
 ---
 
+<br />
+
 ## 1. 🎢 Preparation step
 
-- 😎 Choose a cool name for your project
+<br />
+
+<details>
+<summary>😎 Choose a cool name for your project</summary>
+<p>
+
+> Choose a name users can easily remember. It must not necessarly includes keywords related to the technology you're using within your project (ex: [bento-starter](https://github.com/kefranabg/bento-starter)). 
+
+</p>
+</details>
+
+<details>
+<summary>💅 Make your README pretty</summary>
+<p>
+
+> README is the first your user will see. Make it simple, pretty and easy to read. [Here is a list of beautiful READMEs](https://github.com/matiassingers/awesome-readme).
+
+</p>
+</details>
+
+<details>
+<summary>💪 Highlight strong points of your project </summary>
+<p>
+
+> Identify your project strengths and make sure that visitors see them at first.
+
+</p>
+</details>
+
+<details>
+<summary>✨ Provide a demo of your project</summary>
+<p>
 
-- 💅 Make your README pretty
+> README is the first your user will see. Make it simple, pretty and easy to read. [Here is a list of beautiful READMEs](https://github.com/matiassingers/awesome-readme).
 
-    <details>
-    <summary>Details</summary>
-    <p>
+ </p>
+</details>
 
-  > [List of beautiful readmes](https://github.com/matiassingers/awesome-readme).
+<details>
+<summary>👌 Insall/Usage must be super easy</summary>
+<p>
 
-    </p>
-    </details>
+> Install/Usage must be as simple as possible. You'll probably lost users if your project is not user friendly.
 
-- ✨ Provide a demo of your project
+ </p>
+</details>
 
-- 👌 Insall/Usage must be super easy
+<details>
+<summary>📘 Create a clean and well structured documentation</summary>
+<p>
 
-- 📘 Create a clean and well structured documentation
+> Create a good documentation is probably the most important step. If your have a small documentation, you can include it in your README. Otherwise, you should probably host it in a diferrent website. Projects like [vuepress](https://v1.vuepress.vuejs.org) helps you creating clean documentations in a simple way.
```

Comment:
```suggestion
> It is probably the most important step. If your have a small documentation, you can include it in your README. Otherwise, you should probably host it in a different website. Projects like [vuepress](https://v1.vuepress.vuejs.org) helps you creating clean documentations in a simple way.
```